### ISSUE TITLE
Performance is faster after having joined the human team in plat23

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#2656]
OK, this issue title is very weird. But it's real. This can be seen on old computers, and maybe only on some old ones. I don't know if that's because of GPU being old or because of CPU being old, anyway, it's real, and I reproduced it very reliably at the time.

I noticed this months ago, but haven't reported it yet, as far as I know.

Basically, if I open the plat23 map, and stick on default spectator view, I get `x` amount of fps, then if I join the human  team and immediately leave to be spectator again and stick on the default spectator view, I get `X` amount of fps were `X` is way greater than `x`.

The bug is minor because it affects only spectators (assuming joining alien team also workarounds the bug), but this is very weird to see it in action so that may hide a real bug.

**Comment by necessarily-equal** [comment]
Could this be related to https://github.com/Unvanquished/Unvanquished/issues/2653?

**Comment by illwieckz** [comment]
There was no `vid_restart` involved, and #2653 was caused by the server not sending some data after a `vid_restart`. So, no.

**Comment by necessarily-equal** [comment]
No, but I mean, could it be caused by a similar issue where some 
additional or different pmove data is sent for a team, but not for 
spectator so that you would have some settings to default on loading, 
then those settings wouldn't be set back to those defaults when 
switching to spectator again. In which case you sometimes have the 
initial, and sometimes the humans values as a spectator.

**Comment by illwieckz** [comment]
Ah, well, maybe. Anyway it's very likely that doing that action changes something somewhere.
